.github/workflows/npm-update-pf.yml

```diff
@@ -28,8 +28,17 @@ jobs:
           mkdir -p ~/.config/cockpit-dev
           echo ${{ github.token }} >> ~/.config/cockpit-dev/github-token
           eval $(ssh-agent)
-          ssh-add - <<< '${{ secrets.DEPLOY_KEY }}'
+          ssh-add - <<< '${{ secrets.NODE_CACHE_DEPLOY_KEY }}'
           bots/npm-update @patternfly >&2
           ssh-add -D
           ssh-agent -k
 
+      - name: Force push the change to trigger testing workflows
+        run: |
+          sleep 1 # make sure the timestamp changes
+          git commit --amend --no-edit
+          eval $(ssh-agent)
+          ssh-add - <<< '${{ secrets.COCKPIT_DEPLOY_KEY }}'
+          git push --force 'git@github.com:${{ github.repository }}' HEAD
```

Comment:
There's a wart here: The initial PR creation from bots/npm-update will already queue/start our own tests, as we explicitly allowed the `github-actions` user into our contributors team. Our test machinery is clever enough to abort tests for obsolete PR SHAs, but it does create some noise. We could prevent that by creating the initial PR with no-test and drop the label here.

But this is not a blocker at all -- our current workflow has the same problem (except that tests fail very quickly due to the package.json vs. lock mismatch), so we can do this as an optimization later on if we care enough.

Reply:
OK.